manifest/manifestlist/manifestlist_test.go

```diff
@@ -109,3 +110,105 @@ func TestManifestList(t *testing.T) {
 		}
 	}
 }
+
+// TODO (mikebrow): add annotations on the index and individual manifests
+var expectedOCIImageIndexSerialization = []byte(`{
```

Comment:
Not sure I understand. OCI descriptors are the same as v2 manifest descriptors (with the addition of annotations). In the context of a manifest-list or image index, platform is added which is explicitly optional in the OCI specification, but not explicitly so in the v2 specification. My question was basically how best to use "OCI Manifest descriptors" with optional platform everywhere.